app/ios/Podfile

```diff
@@ -22,6 +22,10 @@ end
 target 'AriesBifold' do
   config = use_native_modules!
 
+  permissions_path = '../node_modules/react-native-permissions/ios'
```

Comment:
Were these two lines added automatically or manually?

Reply:
Manually, following the installation instructions here: https://github.com/zoontek/react-native-permissions#setup